modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -23,6 +23,16 @@ You run a DPDK checkup by performing the following steps:
 * You have installed the OpenShift CLI (`oc`).
 * You have configured the compute nodes to run DPDK applications on VMs with zero packet loss.
 
+[IMPORTANT]
+====
+The traffic generator pod used by this checkup requires elevated privileges:
+
+* It runs as root.
+* It has a bind mount to the node's file system.
```

Comment:
I think you should also specify how the user can obtain these 2 points, i.e. - how to run the pod as root, and how to bind it to the node's FS.

Reply:
The checkup automatically creates the traffic-gen pod this way - running as root and with the bind mount to the node's FS.

Reply:
In that case - you might want to add that it is already done by the job, and no user action is required.
Maybe it's just me (and if anyone who reviews this thinks otherwise - please correct me), but to me this looks like a list of things that the user should take care of.

Reply:
@yossisegev as we've discussed offline, the wording was change.
Is it better now?

Reply:
@orelmisan Indeed, that's clearer now (to me, at least :) ).